sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/EventHubConsumerAsyncClientIntegrationTest.java

```diff
@@ -423,9 +423,10 @@ public void receivesMultiplePartitions() {
 
         // This is the one we'll mutate.
         final Set<Integer> expectedPartitions = new HashSet<>(allPartitions);
+        final Set<Integer> matchPartitions = Collections.synchronizedSet(new HashSet<>(allPartitions));
```

Comment:
Is there a reason why we create another variable? `matchPartitions` is the same as `allPartitions`

Reply:
Fixed in the new version.